src/app/pages/Swap/form/SwapInfoDropdown/index.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { forwardRef, ReactNode } from 'react';
+
+import BigNumber from 'bignumber.js';
+import clsx from 'clsx';
+
+import { Divider, IconBase } from 'app/atoms';
+import { ReactComponent as ArrowDownIcon } from 'app/icons/base/arrow_down.svg';
+import { ReactComponent as GiftIcon } from 'app/icons/base/gift.svg';
+import { ReactComponent as RouteIcon } from 'app/icons/base/route.svg';
+import { ReactComponent as StackIcon } from 'app/icons/base/stack.svg';
+import { ReactComponent as ChevronUpIcon } from 'app/icons/chevron-up.svg';
```

Comment:
this is an old icon, import a new one as `icons/base/chevron_up.svg` and use IconBase to show it